libs/api/domains/education/src/lib/graphql/inna/periods.model.ts

```diff
@@ -20,8 +20,8 @@ export class CoursesModel {
   @Field(() => String, { nullable: true })
   units?: number
 
-  @Field(() => String, { nullable: true })
-  stage?: string
+  @Field(() => Number, { nullable: true })
+  stage?: number | null
```

Comment:
Isn't unnecessary to have it nullable and also null?

Reply:
Yea, fixed